iocBoot/iocEK9KTest/st.tpl

```diff
@@ -18,6 +18,6 @@ cd "${TOP}/iocBoot/${IOC}"
 
 $RECORDS$
 
-dbLoadRecords("../../db/ek9000_status.db", "EK9K=EK9K1")
+dbLoadRecords("../../db/ek9000_status.db", "RECORD=ek9000,EK9K=EK9K1")
```

Comment:
I think `P` or `PREFIX` would be a better/more common macro name for this purpose.

Also, can we set a better default - or perhaps require the user to specify it to generate the startup script?

Reply:
I added some options to `gen-cmd.py` to facilitate this